src/components/journalposter/ForhandsvisningModal.tsx

```diff
@@ -15,14 +13,9 @@ export default function ForhandsvisningModal({
 }): JSX.Element {
   // TODO: Vi må sette appElement på modalen for skjermlesere
```

Comment:
Er dette fortsatt TODO?

Reply:
Jeg ser at den ble lagt til for et år siden. Fjerner den siden vi nå har gått over til en annen modal-løsning.